lib/galaxy/webapps/galaxy/services/quotas.py

```diff
@@ -1,17 +1,14 @@
 import logging
 from typing import Optional
 
-from sqlalchemy import (
-    false,
-    true,
-)
-
-from galaxy import (
-    model,
-    util,
-)
+from galaxy import util
 from galaxy.managers.context import ProvidesUserContext
-from galaxy.managers.quotas import QuotaManager
+from galaxy.managers.groups import get_group_by_name
+from galaxy.managers.quotas import (
+    get_quotas,
```

Comment:
get_quotas is only used here, so keep it here ?

Reply:
Same as with the other services module - ideally, I think, move low-level SA code out of services? Even if it's used only there? Even if it looks trivial (and of course it is), it's still low-level SA code constructing a query against the database. I do feel it should be located separately from the business logic.. If you feel otherwise, I'll certainly change it, and we can return to this topic later, outside of the context of this PR.

Reply:
Can you keep it here ? It's the minimum of refactoring we can agree on.

Reply:
Done.